docs/auth/Single Sign-On via Device Browser.md

```diff
@@ -0,0 +1,27 @@
+## Single Sign-On (SSO) via the Device Browser
+The Auth SDK uses OpenID [AppAuth library](https://github.com/openid/AppAuth-Android) and implements OpenID's [auhtorization code flow](http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth) to achieve SSO via the device browser.
+
+### Prerequisite's
+1. Ensure your app is using the Android SDK - [Getting Started with AeroGear Services Android SDK](https://github.com/aerogear/aerogear-android-sdk/blob/master/docs/getting-started.adoc) and add in the Auth SDK dependency - [AeroGear Services Auth SDD](https://github.com/aerogear/aerogear-android-sdk/blob/master/docs/auth/README.adoc)
```

Comment:
It may be easier to just add gradle snippets here instead of referencing a two external documents.  WDYT?

Reply:
I thought of this too but would we end up just copying the same docs everywhere? I'll have 4 prereq's now - import the Android SDK, import the Core SDK, import the Auth SDK and have a keycloak instance.  I can write out the code snippets for them but I think it'll just bloat the prereq section which has already been documented somewhere else